README.md

```diff
@@ -338,7 +338,7 @@ $ sudo systemctl start mtg
 or you can run a docker image
 
 ```console
-docker run -d -v /etc/mtg.toml:/config.toml -p 443:3128 --name mtg-proxy --restart=unless-stopped nineseconds/mtg:2
+docker run -d -v /etc/mtg.toml:$PWD/config.toml -p 443:3128 --name mtg-proxy --restart=unless-stopped nineseconds/mtg:2
```

Comment:
`$PWD` is specific to your machine so in general this does not work. I guess, this is a typo and you want `-v $PWD/config.toml:config.toml`, am I correct?

Reply:
oh I only meant to add the $PWD

Reply:
Please amend your PR then :)